lib/IDE/ExprContextAnalysis.h

```diff
@@ -1,4 +1,4 @@
-//===--- ExprContextAnalysis.h - Expession context analysis ---------------===//
+//===--- ExprContextAnalysis.h - Expression context analysis ---------------===//
```

Comment:
Could you adjust this to continue being 80 columns?

```suggestion
//===--- ExprContextAnalysis.h - Expression context analysis --------------===//
```